2/scss/about-skout.scss

```diff
@@ -0,0 +1,73 @@
+@import "vars";
+@import "fonts";
+@import "mixins";
+
+.about-skout-container {
+  max-width: 1440px;
+  margin: auto 20px;
+  display: flex;
+  flex-flow: wrap;
+  justify-content: center;
+  flex-direction: column;
+  .skout-sa {
+    order: 2;
+    opacity: 0;
+    transition: opacity 700ms;
+    .skout-sa-img{
+      width: 100%;
+    }
+  }
+  .about-skout {
+    order: 1;
+    opacity: 0;
+    transition: opacity 700ms;
+    max-width: 560px;
+    margin: auto;
+    .title-about-skout {
```

Comment:
`.about-skout-text` - better

```
.about-skout {
  ...
  
  &-container {
    ...
  }

  &-title{
    ...

    &-value {
      ...
    }
  }
}
```
Much more clean and readable code

Reply:
thank you, I understand